tests/unit-tests/unit.test.macros.hh

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "logger.hh"
+
+#define EXPECT(e, ...)                                                         \
```

Comment:
Why don't we use a testing framework like GTest to avoid this boilerplate code every time we configure tests?